rest-api/src/test/java/gov/cms/qpp/conversion/api/services/DbServiceImplTest.java

```diff
@@ -49,11 +48,14 @@
 
 	@BeforeEach
 	void before() {
+		Optional<DynamoDBMapper> dbMapperWrapper = Optional.of(dbMapper);
+		underTest = new DbServiceImpl(taskExecutor, dbMapperWrapper, environment);
 		doAnswer(invocationOnMock -> {
 			Runnable method = invocationOnMock.getArgument(0);
 			CompletableFuture.runAsync(method);
 			return null;
 		}).when(taskExecutor).execute(any(Runnable.class));
+		dbMapper = dbMapperWrapper.get();
```

Comment:
Why do we need to reassign `dbMapper` from the wrapper when just a few lines above this, we create the wrapper from the `dbMapper`?